docs/application/web/tutorials/sec-privileges.md

```diff
@@ -121,7 +123,7 @@ The following tables list the API privileges, which you must declare when using
 <a name="nonAPI"></a>
 ## Non-API Bound Privileges
 
-Tizen application privileges are loosely bound to APIs, so most of the privileges can be identified by the APIs that the application calls. However, there are some privileges that are not coupled with the Tizen APIs. To allow easy identification, those privileges are mapped to corresponding system resources - same as other privileges.
+Tizen application privileges are loosely bound to APIs, so most of the privileges can be identified by the APIs that the application calls. However, there are some privileges that are not coupled with the Tizen APIs. To allow easy identification, those privileges are mapped to corresponding system resources that are similar as other privileges.
```

Comment:
Please modify the last sentence:

 that are similar as other privileges.

to

 that are similar to other privileges.

Reply:
The phrase is reviewed at PR 372 (https://github.com/Samsung/tizen-docs/pull/372) as above and all other sec-privileges.md was written as above. Should I rephrase that? I want to keep consistency between documents.

Reply:
@yl33,

Only to maintain consistency, please ignore this comment.
however, just as a suggestion, we can keep note of it and modify it everywhere later.

Reply:
Question to @surbhig12 and @divyk, who reviewed #372.
Is "~similar as" grammatically correct? It looks like "~similar to" is correct to use.

If we should use "~similar to", @yl33, would you update the pages to follow the grammar rule.
Sorry for the inconvenience.

Reply:
@mijin85cho, 
"Similar to" is correct.

Reply:
@mijin85cho 
"similar to" is grammatically correct as we compare two things and see if it has a resemblance.